README.md

```diff
@@ -107,6 +107,20 @@ scoop reset deno
 
 ## Known Issues
 
+### The program [`unzip`](https://linux.die.net/man/1/unzip) is a requirement for the Shell installer
```

Comment:
Maybe shorten the issue heading to "unzip is required" and move the more detailed explanation into a text block below the heading?

Then you can also link to the known issue from the shell script (https://github.com/denoland/deno_install#unzip-is-required) rather than just printing `unzip is required to install deno`.

What do you think?

Reply:
No problem, thanks for the reviews.